examples/over-sampling/plot_over_sampling_benchmark_3nn.py

```diff
@@ -0,0 +1,97 @@
+"""
+=============================================================
+Benchmark over-sampling methods using a 3NN classifier
+=============================================================
+In this computer vision example two faces are used from the LFW 
```

Comment:
I would not call it a computer vision. This is more pattern recognition or something like:

http://host.robots.ox.ac.uk/pascal/VOC/